modules/setting-up-a-customized-argocd-instance.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started--with-openshift-gitops.adoc
+
+[id="setting-up-a-customized-argocd-instance_{context}"]
+= Setting up a customized Argo CD instance
+
+The Red Hat OpenShift GitOps operator automatically installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions, allowing Argo CD to manage cluster-scoped resources.
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of Argo CD. Using your own instance of Argo CD helps you to continuously deliver your applications.
+
+.Installing ArgoCD
+
+To install an Argo CD instance:
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators*.
+. Create a new project or select an existing project, depending on where you want to install the Argo CD instance.
+. In the *Installed Operators* page, select *OpenShift GitOps* and then the *Argo CD* tab.
+. Create an Argo CD instance and specify the following configurations:
+*** Name: <argocd-instance-name>
+*** Server > Route > Enable Route = true. This configuration creates an external open source (OS) route to access the Argo CD server.
```

Comment:
Server > Route > Enable Route: enclose each of these in asterix, making them bold and use ->